src/PrestaShopBundle/Resources/views/Admin/macros.html.twig

```diff
@@ -142,11 +142,6 @@
     {% endfor %}
 
     {#iterating over child errors because errors can be nested#}
```

Comment:
Reasoning for that is that now I am collecting all the errors from children in TranslatableType so this is no longer needed. As far as I tested translatable types still work and according due commits and comment this was added specifically for translatable type, but I am not 100% sure this won't affect something else.

Reply:
Well, QA will be able to help us asserting the 100% 😉